File: articles/feedback-loops-in-algorithmic-systems/fortran/feedback_risk.f90
program feedback_risk
  implicit none
  real :: amplification, concentration, intervention, drift, recursive_data, score
  amplification = 0.82
  concentration = 0.76
  intervention = 0.44
  drift = 0.28
  recursive_data = 0.31
  score = (amplification + concentration + intervention + drift + recursive_data) / 5.0
  print *, "feedback_risk_score=", score
end program feedback_risk

File: articles/feedback-loops-in-algorithmic-systems/java/FeedbackRisk.java
public class FeedbackRisk {
    public static void main(String[] args) {
        double amplification = 0.82;
        double concentration = 0.76;
        double intervention = 0.44;
        double drift = 0.28;
        double recursiveData = 0.31;
        double score = (amplification + concentration + intervention + drift + recursiveData) / 5.0;
        System.out.printf("feedback_risk_score=%.4f%n", score);
    }
}
